### DISCUSSION TITLE
"A look behind the scenes of <project>" talk

### DISCUSSION THREAD

**Discussion by meisenzahl** [elementary/edw#12] [Ideas]
Speaker: **TBD**

A behind-the-scenes look at the development of a project could be an interesting type of presentation.

For example, it would be interesting to hear the stories of @lxmcf's Virgil and @alecaddd's Akira:
- how did they come up with the idea?
- what milestones have they achieved?
- how do they stay motivated to work on such ambitious projects?

**Comment by Alecaddd** [comment]
I would be honored to prepare a talk for this.
Should I open a new discussion to propose Akira?

**Reply by cassidyjames** [reply]
@Alecaddd yes, I think a new discussion specifically for Akira makes sense.

**Comment by lxmcf** [comment]
Happy to give it a shot :-)

Have raised up a discussion #21
